website/pages/index.tsx

```diff
@@ -1,5 +1,327 @@
+            aria-hidden="true"
+            height="100%"
+            className={css`
+              min-width: 100%;
+            `}
+          />
+          <div className={textWrapper}>{children}</div>
+        </div>
+      </a>
+    </div>
+  );
+}
 
 export default function Home() {
-  return <div>this is the home page</div>;
+  const viewport = useViewportSize();
+  const isTouchDevice = viewport?.width
+    ? viewport?.width < breakpoints.Tablet
+    : false;
+
+  return (
+    <>
+      <div className={backdrop} />
+      <GridContainer
+        role="main"
+        wrap="wrap"
+        align="flex-start"
+        justify="flex-start"
+        className={layoutProperties}
+      >
+        {/* First Row */}
+        <GridItem sm={12} md={6} lg={6}>
+          <div className={lgHeight}>
+            <News />
+          </div>
+        </GridItem>
+        <GridItem sm={6} md={6} lg={6}>
+          <ComponentPreview
+            path="/component/banner/example"
+            background="/images/banner-thumbnail.png"
+            content="Banner"
+            className={lgHeight}
+          />
+        </GridItem>
+
+        {/* Second Row */}
+        {isTouchDevice && (
+          <GridItem sm={6} md={6} lg={6}>
+            <div className={lgHeight}>
+              {/* @ts-expect-error, awaiting URL for marketing page */}
+              <MarketingPreview background="/images/personas-thumbnail.png">
+                Meet our Personas
+              </MarketingPreview>
+            </div>
+          </GridItem>
+        )}
+        <GridItem sm={12} md={6} lg={6}>
+          <div className={secondRowContainer}>
+            <ComponentPreview
+              path="/component/radio-box-group/example"
+              background="/images/radioBox-thumbnail.png"
+              content="Radio boxes"
+              className={cx(smHeight, halfWidth, boxShadow)}
+            />
+            <ComponentPreview
+              path="/component/text-input/example"
+              background="/images/textInput-thumbnail.png"
+              content="Text input"
+              className={cx(smHeight, halfWidth, boxShadow)}
+            />
+            <ComponentPreview
+              path="/component/logo/example"
+              background="/images/logos-thumbnail.png"
+              content="Logos"
+              className={cx(smHeight, halfWidth)}
+            />
+            <ComponentPreview
+              path="/component/tokens/example"
+              background="/images/spacers-thumbnail.png"
+              content="Tokens"
+              className={cx(smHeight, halfWidth)}
+            />
+          </div>
+        </GridItem>
+        {!isTouchDevice && (
+          <GridItem sm={6} md={6} lg={6}>
+            {/* @ts-expect-error, awaiting URL for marketing page */}
+            <MarketingPreview background="/images/personas-thumbnail.png">
+              Meet our Personas
+            </MarketingPreview>
+          </GridItem>
+        )}
```

Comment:
I think it'd be preferable to omit this section or have a meaningful placeholder than have it be non-functional

Reply:
I had this in so that Rob and I could QA, I can omit if you prefer

Reply:
It shouldn't be up to my preference. Rather, we should collectively decide whether we want to display non-functional content in production. I think a link to a coming soon page would be better than a non-functional one.

You may want to look into Feature Flags as a general solution to being able to toggle on functionality only for development.